recipe/meta.yaml

```
@@ -29,13 +34,17 @@ outputs:
         - libtool
         - make
         - pkg-config
+        - libgomp
       host:
-        - cuda-cudart-dev ={{ cuda_compiler_version }}     # [(cuda_compiler_version or "None").startswith("12")]
-        - cuda-nvml-dev ={{ cuda_compiler_version }}       # [(cuda_compiler_version or "None").startswith("12")]
+        - cuda-version {{ cuda_compiler_version }}
+        - cuda-cudart-dev          # [(cuda_compiler_version or "None").startswith("12")]
+        - cuda-nvml-dev            # [(cuda_compiler_version or "None").startswith("12")]
         - rdma-core
       run_constrained:
         - cuda-version >=11.2,<12  # [(cuda_compiler_version or "None").startswith("11")]
         - cuda-version >=12,<13    # [(cuda_compiler_version or "None").startswith("12")]
+        - cudatoolkit              # [(cuda_compiler_version or "None").startswith("11")]
+        - cuda-cudart              # [(cuda_compiler_version or "None").startswith("12")]
```

I think you actually want this
```suggestion
        - cuda-cudart <0a0              # [(cuda_compiler_version or "None").startswith("11")]
        - cudatoolkit <0a0             # [(cuda_compiler_version or "None").startswith("12")]
```

---

Sorry Uwe, but the answer is again no 😅 We've done the mutual exclusion elsewhere properly (both `cudatoolkit` and `cuda-cudart` depend on mutually exclusive set of `cuda-version`s, also `cudatoolkit` is no longer offered/updated starting CUDA 12), so here it's really about declaring optional dependencies (see also the post-link messages that we added).

---

But then you are just adding an empty statement. `run_constraints` is there to set version constraints. Without version constraints, these entries have no effect.

---

We can say that this is no-op. Admittedly the two listed packages here are unorthodox because of the way the actual version constraint (from `cuda-version`) works. From the maintainer perspective, though, an explicit list helps us track the status quo.

---

Can you then add a comment on why you are adding them here? Otherwise, there will be a PR in future that will drop them.

---

Sure, done in commit ddcf7d8.